docs/mine/lotus/split-markets-miners.md

```diff
@@ -126,3 +126,41 @@ export LOTUS_MINER_PATH=~/markets-repo-location
 ```sh
 LOTUS_MINER_PATH=~/markets-repo-location ./lotus-miner run
 ```
+
+## Rollback from a multi-process architecture back to `lotus-miner` monolith process
+
+In case you want to revert the changes listed above and go back to running `lotus-miner` as a single process, you have to do the following:
+
+1. Make sure that the `mining/sealing/proving` node is publicly exposed, as we will be enabling the markets subsystem on it.
+
+2. In the `mining/sealing/proving` repository, update the `config.toml` and set `EnableMarkets` option to `true`
```

Comment:
```suggestion
1. In the `mining/sealing/proving` repository, update the `config.toml` and set `EnableMarkets` option to `true`
```

I'm confused about the `the `mining/sealing/proving` repository` bit. Are you referring to one of these repositories? Or does the reader have to change an option in all three?

Reply:
Yes, I am referring to a specific repository - the one for the main miner node, i.e. the `mining/sealing/proving` node, and NOT the `markets` node.

If the user is attempting a rollback from a multi-process lotus miner architecture, then they have at least 2 repositories at this stage.

Reply:
Oh I see! The `mining/sealing/proving` node is _one node_ that mines, seals, and provides? That makes sense then.